src/components/Personalization/helper/click.js

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { uuid } from "../../../utils";
+import { awaitSelector } from "../../../utils/dom";
+import { selectNodesWithEq, setAttribute } from "./dom";
+
+const ALLOY_KEY = "data-alloy-key";
```

Comment:
I think we already discussed if this needed to be configurable and that it won't be required (until asked for), correct?

Reply:
Yeah, I don't think has to be configurable, but I do think it should probably be part of `constants` so others can access it, thoughts?